### PR TITLE
first commit to divya.md file

### PR DESCRIPTION
**Pull request by Divshub13**
TL;DR this is pr update to update the caption under slides

## Done
- create branch
- made a commit

## To

- [ ]  merge the branch
- [ ] delete the branch

cc: @nicoleescoto

### CODE REVIEW

File: _slides/02-Divshub13.md [hunk 1]
```diff
@@ -3,7 +3,7 @@ layout: slide
 title: "Welcome to our slide deck!"
 ---
 
-Caption here
+This is Divya's-caption
```
Comment (Divshub13) [inline _slides/02-Divshub13.md:6]: This title looks good.

**Review by Divshub13 — COMMENTED** [review]
done

**Comment by nicoleescoto** [comment]
Here is my comment

**Comment by nicoleescoto** [comment]
Here is my second comment

File: _slides/02-Divshub13.md [hunk 2]
```diff
@@ -3,7 +3,7 @@ layout: slide
 title: "Welcome to our slide deck!"
 ---
```
Comment (williamkao1928) [inline _slides/02-Divshub13.md:4]: this is good

**Pull request merged**